docs/overview.rst

```diff
@@ -209,12 +209,6 @@ or our :ref:`tutorials <tutorials>`!
 - `kinto-http-java <https://github.com/intesens/kinto-http-java>`_: A Java HTTP Client
 - `ember-kinto <https://github.com/ptgamr/ember-kinto>`_: Offline first Ember Data adapter
 
-**Coming soon**
-
-- Push notifications using `the Push API <https://developer.mozilla.org/en-US/docs/Web/API/Push_API>`_ (:github:`under construction <Kinto/kinto-webpush>`)
-
```

Comment:
Ok, if webpush-channels replace kinto-webpush, then we need a documentation page that explains how to receive push notifications when some record is changed in Kinto